clippy_lints/src/len_zero.rs

```diff
@@ -226,23 +239,23 @@ impl<'tcx> LateLintPass<'tcx> for LenZero {
             let actual_span = span_without_enclosing_paren(cx, expr.span);
             match cmp {
                 BinOpKind::Eq => {
-                    check_cmp(cx, actual_span, left, right, "", 0); // len == 0
-                    check_cmp(cx, actual_span, right, left, "", 0); // 0 == len
+                    check_cmp(cx, actual_span, left, right, "", 0, self.msrv); // len == 0
```

Comment:
Those functions would probably deserve to be methods in order to get the MSRV from `self`.